### PR TITLE
Raster gtm hotfix

### PR DESCRIPTION
**Pull request by gfiorav**
Please review @juanignaciosl

### CODE REVIEW

File: lib/carto/ghost_tables_manager.rb [hunk 1]
```diff
@@ -132,10 +135,18 @@ def unaltered?
       !new? && !renamed? && !regenerated?
     end
 
+    def raster?
+      user_table.table_visualization.raster_kind? if user_table && user_table.table_visualization
```
Comment (juanignaciosl) [inline lib/carto/ghost_tables_manager.rb]: Simpler suggestion:

```ruby
user_table && user_table.table_visualization && user_table.table_visualization.raster_kind?
```

Comment (juanignaciosl) [inline lib/carto/ghost_tables_manager.rb]: Correction: if you reuse `user_table` you'll query too many times. Proposal:

```ruby
def raster?
  ut = user_table
  ut && ut.table_visualization && ut.table_visualization.raster_kind?
end

Comment (juanignaciosl) [inline lib/carto/ghost_tables_manager.rb]: ... and more or less the same with `table_visualization`. Complete suggestion:

```ruby
def raster?
  v = visualization
  v && v.raster_kind?
end

def visualization
  ut = user_table
  ut ? ut.table_visualization : nil
end
```

**Comment by juanignaciosl** [comment]
I've made some comments to reduce the amount of database querying. 👍 afterwards.

**Comment by gfiorav** [comment]
See if it's ok now! Thanks!

File: lib/carto/ghost_tables_manager.rb [hunk 2]
```diff
@@ -132,10 +135,21 @@ def unaltered?
       !new? && !renamed? && !regenerated?
     end
 
+    def raster?
+      ut = user_table
+      ut_visualization = ut.nil? ? nil : ut.table_visualization
+
+      ut_visualization.raster_kind? unless !ut_visualization
```
Comment (juanignaciosl) [inline lib/carto/ghost_tables_manager.rb]: `ut_visualization ? ut_visualization.raster_kind? : false`

or

`ut_visualization.raster_kind? if ut_visualization`

is simpler

**Comment by juanignaciosl** [comment]
Added a last comment mostly because of having a `unless !`, but up to you.

**Comment by gfiorav** [comment]
Oh, I'm going fast an making these silly overlooks 😓 

I've changed it, once old test go green I'll deploy.

**Pull request merged**